workflows/kube-proxy-all/workflow_cron.yaml

```diff
@@ -3,6 +3,7 @@ kind: CronWorkflow
 metadata:
   name: argo-chaos-kube-proxy-cron-wf
   namespace: litmus
+  subject: litmus-kube-proxy
```

Comment:
please use kube-system/kube-proxy

Reply:
We can use ```kube-system_kube-proxy``` since k8s label don't allow to us ```/```